### DISCUSSION TITLE
Socio.js similar project

### DISCUSSION THREAD

**Discussion by Rolands-Laucis** [akash-joshi/rocketrpc#20] [Show and tell]
Heyo, saw your presentation on Svelte Society Youtube channel and had to investigate your project. After looking into it, I see we have a similar view of the future of the Web, as I am also making a WebSocket based API framework - ["Socio"](https://github.com/Rolands-Laucis/Socio) . 

I'm just friendly saying "hello" and endorse your project :) I will be adding a link to this repo in my project, because more people need to know about these new technologies. And currently it is difficult to find our projects when googling for such a thing. Lets grow the Web!

**Comment by akash-joshi** [comment]
@Rolands-Laucis this looks great! Why don't you expand this to support all kinds of objects? Maybe there's a doc that I'm missing?

**Reply by Rolands-Laucis** [reply]
Im not sure, what you mean. Anything JSON serializable can be sent either direction. I also added serialization logic for js Map and Set objects.
